query_execution/PolicyEnforcerBase.hpp

```diff
@@ -179,6 +180,18 @@ class PolicyEnforcerBase {
    **/
   virtual bool admitQuery(QueryHandle *query_handle) = 0;
 
+#ifdef QUICKSTEP_DISTRIBUTED
```

Comment:
Same comment as in PolicyEnforcerBase.cpp. Why are we using this macro in the base class? The base class can define virtual functions, as we wish. The single node derived implementation can be a no-op or through an error if the function concerns only with the distributed implementation.